index.html

```diff
@@ -671,6 +681,10 @@ <h4>Redirected and Proxied External Content</h4>
             <code>GET</code> and <code>HEAD</code> requests to any external <a>LDP-NR</a> MUST correctly
             respond to the <code>Want-Digest</code> header defined in [[!RFC3230]].
           </p>
+          <p id='external-content-redirect-status'>
+            A successful response to external content with <code>handling</code> of <code>redirect</code> MUST have
```

Comment:
response to **a <code>GET</code> and <code>HEAD</code> request** for external content?